Source/WebCore/Modules/webauthn/WebAuthenticationConstants.h

```diff
@@ -83,7 +83,9 @@ enum class ShouldZeroAAGUID : bool {
     Yes
 };
 
-constexpr const char LocalAuthenticatorAccessGroup[] = "com.apple.webkit.webauthn";
+#if defined(__OBJC__)
+static NSString* const LocalAuthenticatorAccessGroup = @"com.apple.webkit.webauthn";
```

Comment:
This should only be a declaration in a header with a definition in a .mm file.

Reply:
The use of `static` is wrong here